curriculum/challenges/english/09-information-security/information-security-projects/anonymous-message-board.md

```diff
@@ -8,21 +8,43 @@ forumTopicId: 301568
 ## Description
 <section id='description'>
 Build a full stack JavaScript app that is functionally similar to this: <a href="https://anonymous-message-board.freecodecamp.rocks/" target="_blank">https://anonymous-message-board.freecodecamp.rocks/</a>.
-Working on this project will involve you writing your code on Repl.it on our starter project. After completing this project you can copy your public Repl.it URL (to the homepage of your app) into this screen to test it! Optionally you may choose to write your project on another platform but it must be publicly visible for our testing.
-Start this project on Repl.it using <a href="https://repl.it/github/freeCodeCamp/boilerplate-project-messageboard">this link</a> or clone <a href='https://github.com/freeCodeCamp/boilerplate-project-messageboard/'>this repository</a> on GitHub! If you use Repl.it, remember to save the link to your project somewhere safe!
+Working on this project will involve you writing your code using one of the following methods:
+
+- Clone <a href='https://github.com/freeCodeCamp/boilerplate-project-messageboard/'>this GitHub repo</a> and complete your project locally.
+- Use <a href='https://repl.it/github/freeCodeCamp/boilerplate-project-messageboard' target='_blank'>our repl.it starter project</a> to complete your project.
```

Comment:
The Repl link is opened in a new tab, while the GitHub one does not. I'm wondering if this is intentional

Reply:
@huyenltnguyen Good catch!  Would you be interested in updating all the backend project's with the `_blank` for the GitHub link in a separate PR?

Reply:
I'll get this fixed on this PR. 🙂

Reply:
@huyenltnguyen Actually, I decided to go ahead and not only fix the other such project links, but also make sure they are using the new `description` we are using.  I should have a PR shortly.